### PR TITLE
Nakul/edit data mysql

### PR DESCRIPTION
**Pull request by nakulms**

### CODE REVIEW

File: ossdbtoolsservice/edit_data/templates/pg_templater.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from .templater import Templater
```
Comment (rishky-msft) [inline ossdbtoolsservice/edit_data/templates/pg_templater.py:5]: Where this template existing previously?

File: ossdbtoolsservice/object_explorer/routing/mysql_routing.py [hunk 2]
```diff
@@ -45,6 +45,9 @@ def _get_node_info(
     # Generate the object metadata
     metadata = ObjectMetadata(node.urn, None, type(node).__name__, node.name, None)
 
+    if hasattr(node, '_dbname') and node._dbname:
```
Comment (rishky-msft) [inline ossdbtoolsservice/object_explorer/routing/mysql_routing.py:48]: Why is this required?

File: ossdbtoolsservice/parsers/datatype_parsers.py [hunk 3]
```diff
@@ -65,7 +67,9 @@ def parse_time_with_timezone(value: str) -> datetime.time:
 
 
 def parse_datetime(value: str) -> datetime.datetime:
-    if value == 'now()':
+    if value == 'now()':  # PG
+        return datetime.datetime.now()
+    elif value == 'CURRENT_TIMESTAMP':  # MySQL
```
Comment (rishky-msft) [inline ossdbtoolsservice/parsers/datatype_parsers.py:72]: With this change both values with work for both providers.
Can we check if it is requried?

File: ossdbtoolsservice/parsers/mysql_datatypes.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+DATATYPE_FLOAT = 'float'
+DATATYPE_DOUBLE = 'double'
+DATATYPE_BIT = 'bit'
+
+''' Character Types '''
+DATATYPE_CHAR = 'char'
+DATATYPE_VARCHAR = 'varchar'
+DATATYPE_BINARY = 'binary'
+DATATYPE_VARBINARY = 'varbinary'
+DATATYPE_BLOB = 'blob'
```
Comment (rishky-msft) [inline ossdbtoolsservice/parsers/mysql_datatypes.py:29]: Additional datatypes are also available.
Example : TINYBLOB, etc

**Review by rishky-msft — CHANGES_REQUESTED** [review]

File: ossdbtoolsservice/edit_data/templates/pg_templater.py (hunk 1 above)
Comment (nakulms) [reply ossdbtoolsservice/edit_data/templates/pg_templater.py:5]: Earlier they were using as it is in implementation. Now all the templates are collected in a templater class which is being called to get the template format.

File: ossdbtoolsservice/object_explorer/routing/mysql_routing.py (hunk 2 above)
Comment (nakulms) [reply ossdbtoolsservice/object_explorer/routing/mysql_routing.py:48]: If node has dbname attribute, pass into metadata as schema name. dbname and schema name are the same in MySQL. Added comment in code as well

File: ossdbtoolsservice/parsers/datatype_parsers.py (hunk 3 above)
Comment (nakulms) [reply ossdbtoolsservice/parsers/datatype_parsers.py:72]: modified

File: ossdbtoolsservice/parsers/mysql_datatypes.py (hunk 4 above)
Comment (nakulms) [reply ossdbtoolsservice/parsers/mysql_datatypes.py:29]: BLOB datatypes are not editable. I have added parser for JSON datatype in the latest commit.

**Review by rishky-msft — APPROVED** [review]
